apps/ledger-live-mobile/src/components/RootNavigator/types/PtxNavigator.ts

```diff
@@ -0,0 +1,23 @@
+import { ScreenName } from "~/const";
+
+type CommonParams = {
+  goToURL?: string;
+  lastScreen?: string;
+  name?: string;
+  platform?: string;
+  referrer?: string;
+};
+
+type ExchangeParams = CommonParams & {
+  account?: string;
+  currency?: string; // Used for the deeplink only
+  defaultAccountId?: string;
+  defaultCurrencyId?: string;
+  defaultTicker?: string;
+};
```

Comment:
This type is exactly the same type used in `apps/ledger-live-mobile/src/components/RootNavigator/types/PtxLiveAppNavigator.ts`

Re-use one

Reply:
Unified